<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetSurfaceMeshRenderer</class>
 <widget class="QWidget" name="WidgetSurfaceMeshRenderer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>195</width>
    <height>224</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Surface mesh renderer</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Phong Shading</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="checkBoxPhongShading">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Check to enable Phong shading</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="2" colspan="3">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Color Property</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="checkBoxUseColorProperty">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Check to enable Phong shading</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="2" colspan="3">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>Faces</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QCheckBox" name="checkBoxShowFaces">
       <property name="toolTip">
        <string>Show/Hide surface</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QToolButton" name="toolButtonFacesDefaultColor">
       <property name="toolTip">
        <string>Default surface color</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3" colspan="2">
      <widget class="QSlider" name="horizontalSliderFacesOpacity">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>The opacity of the surface</string>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>60</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Vertices</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QCheckBox" name="checkBoxShowVertices">
       <property name="toolTip">
        <string>Show/Hide vertices</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QToolButton" name="toolButtonVerticesDefaultColor">
       <property name="toolTip">
        <string>Vertices color</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QToolButton" name="toolButtonVerticesImpostors">
       <property name="toolTip">
        <string>Render vertices as spheres</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>Resources/icons/imposters.png</normaloff>Resources/icons/imposters.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxVerticesSize">
       <property name="toolTip">
        <string>Vertices size</string>
       </property>
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="minimum">
        <double>1.000000000000000</double>
       </property>
       <property name="value">
        <double>5.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Wireframe</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QCheckBox" name="checkBoxShowWireframe">
       <property name="toolTip">
        <string>Show/Hide wireframe</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QToolButton" name="toolButtonWireframeDefaultColor">
       <property name="toolTip">
        <string>Wireframe color</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QToolButton" name="toolButtonWireframeImpostors">
       <property name="toolTip">
        <string>Render wireframe as cylinders</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>Resources/icons/imposters.png</normaloff>Resources/icons/imposters.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="4">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxWireframeThickness">
       <property name="toolTip">
        <string>Wireframe thickness (valid when rendered as cylinders)</string>
       </property>
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="minimum">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Borders</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QCheckBox" name="checkBoxShowBorders">
       <property name="toolTip">
        <string>Show/Hide surface borders</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QToolButton" name="toolButtonBordersDefaultColor">
       <property name="toolTip">
        <string>Borders color</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QToolButton" name="toolButtonBordersImpostors">
       <property name="toolTip">
        <string>Render borders as cylinders</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>Resources/icons/imposters.png</normaloff>Resources/icons/imposters.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="4">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxBordersThickness">
       <property name="toolTip">
        <string>Border thickness (valid when rendered as cylinders)</string>
       </property>
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="minimum">
        <double>1.000000000000000</double>
       </property>
       <property name="value">
        <double>2.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_19">
       <property name="text">
        <string>Face Labels</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QCheckBox" name="checkBoxShowFaceLabels">
       <property name="toolTip">
        <string>Show/Hide face lables</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="6" column="2" colspan="3">
      <spacer name="horizontalSpacer_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_20">
       <property name="text">
        <string>Vertex Labels</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QCheckBox" name="checkBoxShowVertexLabels">
       <property name="toolTip">
        <string>Show/Hide vertex labels</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="7" column="2" colspan="3">
      <spacer name="horizontalSpacer_8">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>36</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>checkBoxPhongShading</tabstop>
  <tabstop>checkBoxUseColorProperty</tabstop>
  <tabstop>checkBoxShowFaces</tabstop>
  <tabstop>toolButtonFacesDefaultColor</tabstop>
  <tabstop>horizontalSliderFacesOpacity</tabstop>
  <tabstop>checkBoxShowVertices</tabstop>
  <tabstop>toolButtonVerticesDefaultColor</tabstop>
  <tabstop>toolButtonVerticesImpostors</tabstop>
  <tabstop>doubleSpinBoxVerticesSize</tabstop>
  <tabstop>checkBoxShowWireframe</tabstop>
  <tabstop>toolButtonWireframeDefaultColor</tabstop>
  <tabstop>toolButtonWireframeImpostors</tabstop>
  <tabstop>doubleSpinBoxWireframeThickness</tabstop>
  <tabstop>checkBoxShowBorders</tabstop>
  <tabstop>toolButtonBordersDefaultColor</tabstop>
  <tabstop>toolButtonBordersImpostors</tabstop>
  <tabstop>doubleSpinBoxBordersThickness</tabstop>
  <tabstop>checkBoxShowFaceLabels</tabstop>
  <tabstop>checkBoxShowVertexLabels</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
